packages/labs/cli/src/lib/commands/localize/build.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {Command, flags} from '@oclif/command';
+import {readConfigFileAndWriteSchema} from '@lit/localize-tools/lib/config.js';
+import {TransformLitLocalizer} from '@lit/localize-tools/lib/modes/transform.js';
+import {RuntimeLitLocalizer} from '@lit/localize-tools/lib/modes/runtime.js';
+import type {Config} from '@lit/localize-tools/src/types/config';
+import {KnownError, unreachable} from '@lit/localize-tools/lib/error.js';
+import type {
+  TransformOutputConfig,
+  RuntimeOutputConfig,
+} from '@lit/localize-tools/src/types/modes.js';
+import {LitLocalizer} from '@lit/localize-tools/lib/index.js';
+
+export default class LocalizeBuild extends Command {
+  static description = 'Build lit-localize projects';
+
+  static examples = [`$ lit localize:build`];
```

Comment:
I think `lit localize build` would be more intuitive, with e.g. `lit help localize` showing the available subcommands. I guess ocliff doesn't support this?